### ISSUE TITLE
annotations do not parse negative ints

### ISSUE THREAD

**Issue by georgejecook** [rokucommunity/brighterscript#287]
```
 it.only('params test with negative numbers', () => {
            program.addOrReplaceFile('source/test.spec.bs', `
                @suite
                class ATest
                    @describe("groupA")
                    @it("is test1")
                    @params(100,100)
                    @params(100,-100)
                    @params(-100,100)
                    @params(-100,-100)
                    function Test_3(a, b)
                    end function
                end class
            `);
            program.validate();
            expect(program.getDiagnostics()).to.be.empty;
            expect(plugin.session.sessionInfo.testSuitesToRun).to.not.be.empty;
        });
```

in this scenario, all th -100's turn into null, because the call expression arg is a unary, not a literal expression.

**Comment by georgejecook** [comment]
I've fixed this..
I'll get a pr up some time tomorrow, with an imrpoved unit test (this came from my rooibos plugin, where Ithought the problem originated from )..

fix is this; so if you concur, please don't wate time on it - I only need a few mins to move my test over.
![image](https://user-images.githubusercontent.com/1214545/105611993-308fde80-5db9-11eb-9f61-afa2c0a248a4.png)

**Comment by TwitchBronBron** [comment]
This was fixed in #288
